### ISSUE TITLE
Adding new option to save seed phrase locally in file

### ISSUE THREAD

**Issue by reasje** [MXCzkEVM/datadash-wallet#203]
For Create Wallet we already have three options 
Email, Wechat and Telegram

In case all these three options are disabled - means users dosent have these in their devices

Then we should enable another option to save file locally in theor device and enable user to create wallet

Note: save file locally will only be enabled when other three options are disabled.

**Comment by reasje** [comment]
I have not been able to get the right permisisons and save the file downloads folder 
On IOS the download may not exist so adding that might fix the problem 
On Android I am getting errors trying to save in Downloads folder

**Comment by reasje** [comment]
Android save to downloads is working, IOS save to downloads only saves to to app directory downloads not the general one.

**Comment by reasje** [comment]
IOS doesn't let save files to general Downloads folder 
IOS seed phase should be saved in app directory 
For importing from there I need to look for a solution to open that directory

**Comment by reasje** [comment]
Export and create wallet is done. 
Import wallet with these options are being tested on each platform 
url launcher 
open file 
File picker 
open file manager

**Comment by reasje** [comment]
@SanghamitraBhowmick1993 
I just merged the feat to pre main qa 
1. In my case IOS simulator is working fine for testing but Android Simulator had problems 
2. We will need to run translations before moving to release 
3. If you have any suggestions for UI/UX improvements, I would be glad to hear

**Comment by reasje** [comment]
@SanghamitraBhowmick1993 
 I have enabled test mode and local backup should show up if any option is available 
After testing I will revert that.

**Comment by SanghamitraBhowmick1993** [comment]
For Android, creating a wallet works, but importing does not. The file is saved successfully but fails to import. The same process works on iPhone. I have attached a video demonstrating the issue.



https://github.com/MXCzkEVM/datadash-wallet/assets/93600163/af4e0019-4fe9-4a0b-8ea4-92518a1b5f9e

**Comment by reasje** [comment]
@SanghamitraBhowmick1993 `My Files` was the default selected option for me and It always works for me  😅, Plz check this option, If `My Files`  works I will check why `Files` is not working and report here.

**Comment by SanghamitraBhowmick1993** [comment]
On my phone, 'Files' and 'My Files' are the same thing. When I save a file locally, it gets saved in 'My Files > Downloads. @reasje

**Comment by SanghamitraBhowmick1993** [comment]
yes works with 'My Files' but does not work with 'Files' on android

**Comment by SanghamitraBhowmick1993** [comment]
we still have to disable the local file option pre-release

**Comment by reasje** [comment]
@SanghamitraBhowmick1993 Disabled local seed phrase backup.

**Comment by SanghamitraBhowmick1993** [comment]
added Files issue to this ticket https://github.com/orgs/MXCzkEVM/projects/4/views/1?pane=issue&itemId=65352538
Hence closing this one
